cmake/modules/AddSwift.cmake

```diff
@@ -948,34 +964,38 @@ function(_add_swift_library_single target name)
 
   # Configure plist creation for OS X.
   set(PLIST_INFO_PLIST "Info.plist" CACHE STRING "Plist name")
-  if(APPLE AND SWIFTLIB_SINGLE_IS_STDLIB)
-    set(PLIST_INFO_NAME ${name})
-    set(PLIST_INFO_UTI "com.apple.dt.runtime.${name}")
-    set(PLIST_INFO_VERSION "${SWIFT_VERSION}")
-    if (SWIFT_COMPILER_VERSION)
-      set(PLIST_INFO_BUILD_VERSION
-        "${SWIFT_COMPILER_VERSION}")
-    endif()
+  if(SWIFTLIB_SINGLE_IS_STDLIB)
```

Comment:
Can you move this linker code somewhere else. My reasoning is that appending -shared to the link_flags on android has nothing to do with plist creation. I would look for where linker flags are added for other platforms and put the linker flag change there.